WordPress/Sniffs/WP/DeprecatedParametersSniff.php

```diff
@@ -0,0 +1,349 @@
+	 *            )
+	 *         )
+	 *    );
+	 */
+	protected $target_functions = array(
+
+		'add_option' => array(
+			3 => array(
+				'value'   => '',
+				'version' => '2.3.0',
+			),
+		),
+		'comments_link' => array(
+			1 => array(
+				'value'   => '',
+				'version' => '0.72',
+			),
+			2 => array(
+				'value'   => '',
+				'version' => '1.3.0',
+			),
+		),
+		'comments_number' => array(
+			4 => array(
+				'value'   => '',
+				'version' => '1.3.0',
+			),
+		),
+		'convert_chars' => array(
+			2 => array(
+				'value'   => '',
+				'version' => '0.71',
+			),
+		),
+		'discover_pingback_server_uri' => array(
+			2 => array(
+				'value'   => '',
+				'version' => '2.7.0',
+			),
+		),
+		'get_category_parents' => array(
+			5 => array(
+				'value'   => array(),
+				'version' => '4.8.0',
+			),
+		),
+		'get_delete_post_link' => array(
+			2 => array(
+				'value'   => '',
+				'version' => '3.0.0',
+			),
+		),
+		'get_last_updated' => array(
+			1 => array(
+				'value'   => '',
+				'version' => '3.0.0', // Was previously part of MU.
+			),
+		),
+		'get_the_author' => array(
+			1 => array(
+				'value'   => '',
+				'version' => '2.1.0',
+			),
+		),
+		'get_user_option' => array(
+			3 => array(
+				'value'   => '',
+				'version' => '2.3.0',
+			),
+		),
+		'get_wp_title_rss' => array(
+			1 => array(
+				'value'   => '&#8211;',
+				'version' => '4.4.0',
+			),
+		),
+		'is_email' => array(
+			2 => array(
+				'value'   => false,
+				'version' => '3.0.0',
+			),
+		),
+		'load_plugin_textdomain' => array(
+			2 => array(
+				'value'   => false,
+				'version' => '2.7.0',
+			),
+		),
+		'safecss_filter_attr' => array(
+			2 => array(
+				'value'   => '',
+				'version' => '2.8.1',
+			),
+		),
+		'the_attachment_link' => array(
+			3 => array(
+				'value'   => false,
+				'version' => '2.5.0',
+			),
+		),
+		'the_author' => array(
+			1 => array(
+				'value'   => '',
+				'version' => '2.1.0',
+			),
+			2 => array(
+				'value'   => true,
+				'version' => '1.5.0',
+			),
+		),
+		'the_author_posts_link' => array(
+			1 => array(
+				'value'   => '',
+				'version' => '2.1.0',
+			),
+		),
+		'trackback_rdf' => array(
+			1 => array(
+				'value'   => '',
+				'version' => '2.5.0',
+			),
+		),
+		'trackback_url' => array(
+			1 => array(
+				'value'   => true,
+				'version' => '2.5.0',
+			),
+		),
+		'update_blog_option' => array(
+			4 => array(
+				'value'   => null,
+				'version' => '3.1.0',
+			),
+		),
+		'update_blog_status' => array(
+			4 => array(
+				'value'   => null,
+				'version' => '3.1.0',
+			),
+		),
+		'update_user_status' => array(
+			4 => array(
+				'value'   => null,
+				'version' => '3.0.2',
+			),
+		),
+		'unregister_setting' => array(
+			4 => array(
+				'value'   => '',
+				'version' => '4.7.0',
+			),
+		),
+		'wp_get_http_headers' => array(
+			2 => array(
+				'value'   => false,
+				'version' => '2.7',
```

Comment:
Does this value require being exactly `2.7` (as opposed to `2.7.0`)? If so, maybe worth adding a comment so it's not accidentally "fixed" in the future?